core-web/apps/dotcms-ui/src/app/portlets/dot-containers/container-list/container-list.component.ts

```diff
@@ -132,7 +135,13 @@ export class ContainerListComponent implements OnDestroy {
     getContainersWithDisabledEntities(containers: DotContainer[]): DotContainer[] {
         return containers.map((container) => {
             container.disableInteraction =
-                container.identifier.includes('/') || container.identifier === 'SYSTEM_CONTAINER';
+                container.identifier.includes('/') ||
+                container.identifier === 'SYSTEM_CONTAINER' ||
+                container.source === CONTAINER_SOURCE.FILE;
+
+            if (container.path) {
+                container.pathName = new URL(`http:${container.path}`).pathname;
+            }
```

Comment:
Don't mutate the container object, create a new one.

Reply:
Noted

Reply:
Updated